src/articles/_posts/2009-03-06-javascript-best-practices.md

```diff
@@ -546,26 +547,27 @@ As I am using the generic — really throw-away — variable names `ul` and `li`
 
 	function renderProfiles(o) {
 		var out = document.getElementById('profiles');
+		var ul = document.createElement('ul');
 		for(var i = 0; i < o.members.length; i++) {
-			var ul = document.createElement('ul');
 			var li = document.createElement('li');
-			li.appendChild(document.createTextNode(data.members[i].name));
-			li.appendChild(addMemberData(o.members[i]));
+			li.appendChild(document.createTextNode(o.members[i].name));
+			li.appendChild(addMemberData(o.members[i].data));
```

Comment:
Information hiding and access optimisation (q.v. avoid excessive member access implied by Optimize loops).
Only the data member is used within `addMemberData` so don't supply whole member object and repeatedly access the data member, just supply the data member itself.